lib/src/types.dart

```diff
@@ -64,5 +64,9 @@ enum PostgreSQLDataType {
   ///
   /// Must contain 32 hexadecimal characters. May contain any number of '-' characters.
   /// When returned from database, format will be xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx.
-  uuid
+  uuid,
+
+  /// Must be a [List] of [int] currently in Ewkb(Extended Well-Known Binary) format
```

Comment:
Please link to the spec/documentation of this format.

Reply:
https://postgis.net/docs/using_postgis_dbmanagement.html
I'll add it into the code and push the change

Reply:
e23a3c6 I hope that helps